lib/vagrant/box.rb

```diff
@@ -173,7 +173,7 @@ def repackage(path)
 
       Util::SafeChdir.safe_chdir(@directory) do
         # Find all the files in our current directory and tar it up!
-        files = Dir.glob(File.join(".", "**", "*"))
+        files = Dir.glob(File.join(".", "**", "*")).select { |f| File.file?(f) }
```

Comment:
I don't remember exactly, but I think there was a bug/feature in Ruby - can you verify this includes executables too? You may need to add a `File.executable?` check too, but I could be mis-remembering.